### DISCUSSION TITLE
[Projects Beta] Search icon scrolls off screen

### DISCUSSION THREAD

**Discussion by jsunsawyer** [community/community#6797] [Projects and Issues]
When horizontally scrolling, the search icon scrolls off the screen, making it inaccessible.

This could be improved by moving the search icon to a location that is always visible, or forcing it to maintain its position (which may result in some unwanted overlaps).

**Comment by timoyates** [comment]
Hey @jsunsawyer, thanks for the feedback! This is an excellent point. I'll take this back to the team and see what we can do.
